starters/lit-mobx-vaadin/src/components/counter/counter.spec.ts

```diff
@@ -0,0 +1,88 @@
+import { elementUpdated, expect, fixture, html } from '@open-wc/testing';
+
+import { starterState } from '../../state.js';
```

Comment:
This is a nitpick feel free to ignore.

Maybe we should use an alias for these kind of imports here? Like:

```ts
import { starterState } from '@/state.js';
```

Reply:
I'm skipping these atm because tsc does not handle aliases by itself.